docs/api.yml

```diff
@@ -0,0 +1,270 @@
+            Return object containting list of worker states in workers property
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  name:
+                    type: string
+                  jobs:
+                    type: array
+                    description: |
+                      Default ordering is oldest entries first, except for the
+                      `[failed]` and `[finished]` registries, where it is newest first.
+                    
+                    items:
+                      type: object
+                      required: 
+                        - id
+                        - status
+                        - enqueued_at
+                        - started_at
+                        - ended_at
+                        - origin
+                        - error_message
+                        - description
+                      properties:
+                        id:
+                          type: string
+                          format: uuid
+                        status:
+                          type: string
+                        enqueued_at:
+                          type: string
+                          format: date-time
+                        started_at:
+                          type: string
+                          format: date-time
+                        ended_at:
+                          type: string
+                          format: date-time
+                        origin:
+                          type: string
+                        error_message:
+                          type: string
+                        description:
+                          type: string
+                        worker:
+                          type: string
+                          description: |
+                            Description of Worker that the job is being executed on.
+                            Only present for running jobs (aka in `[running]` registry)
+                  pagination:
+                    type: object
+                    properties:
+                      pages_in_window:
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            number:
+                              type: integer
+                            url:
+                              type: string
+                              format: url
+                      next_page:
+                        type: object
+                        properties:
+                          url:
+                            type: string
+                            format: url
+                      prev_page:
+                        type: object
+                        properties:
+                          url:
+                            type: string
+                            format: url
```

Comment:
```suggestion
                      next_page:
                        type: object
                        nullable: true
                        description: |
                          Despite the naming actually provides a link to the LAST page.
                          On the last page 'null' is used instead of an object.
                        properties:
                          url:
                            type: string
                            format: url
                      prev_page:
                        type: object
                        nullable: true
                        description: |
                          Despite the naming actually provides a link to the FIRST page.
                          On the first page 'null' is used instead of an object.
                        properties:
                          url:
                            type: string
                            format: url
```